server/apps/distributed-app/docs/modules/ROOT/pages/configure/jmap.adoc

```diff
@@ -58,13 +58,15 @@ This enables a higher resilience, but the projection needs to be correctly popul
 
 | authentication.strategy.draft
 | Optional List[String] with delimiter `,` . Specify which authentication strategies system admin want to use for JMAP draft server.
-If non is specified, JMAP draft server will fallback to default strategies:
+The implicit package name is `org.apache.james.jmap.http`. If you have a custom authentication strategy outside this package, you have to specify its FQDN.
+If non authentication strategy is specified, JMAP draft server will fallback to default strategies:
 `AccessTokenAuthenticationStrategy`, `JWTAuthenticationStrategy`, `QueryParameterAccessTokenAuthenticationStrategy`
 
 | authentication.strategy.rfc8621
 | Optional List[String] with delimiter `,` . Specify which authentication strategies system admin want to use for JMAP RFC-8621 server.
-If non is specified, JMAP RFC-8621 server will fallback to default strategies:
-`JWTAuthenticationStrategy`, `BasicAuthenticationStrategy`
+The implicit package name is `org.apache.james.jmap.http`. If you have a custom authentication strategy outside this package, you have to specify its FQDN.
+If non authentication strategy is specified, JMAP RFC-8621 server will fallback to default strategies:
```

Comment:
```suggestion
If no authentication strategy is specified, JMAP RFC-8621 server will fallback to default strategies:
```

Reply:
Ah yes this makes more sense actually thanks :D